frontend/app/helpers/eq.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
```

Comment:
Remove this helper. And instead of this, add this addon https://github.com/jmurphyau/ember-truth-helpers